input/fsh/PatientIdentifierPatternConformanceSE.fsh

```diff
@@ -0,0 +1,29 @@
+Profile: PatientIdentifierPatternConformanceSE
+Parent: PatientSEVendorLite
+Id: PatientIdentifierPatternConformanceSE
+Description: "This profile enables format limitations to be put on the identifier in order to comply with Swedish legislation."
+* ^url = "http://care.commonprofiles.fhir/StructureDefinition/PatientIdentifierPatternConformanceSE"

```

Comment:
url should not be defined manually. A correct url will be generated from global canonical url and profile: id.
This is the only thing that NEEDS to be corrected

Reply:
Sounds like we should fix it then!
Want me to do it or are you feeling fishy?